eZ Platform/Admin UI/REQUIREMENTS.md

```diff
@@ -0,0 +1,45 @@
+# v2 Hybrid Prototype Admin UI Requirements
+## https://github.com/ezsystems/specs/issues/4
+
+# Objective
+"As an eZ Engineer, I want...
```

Comment:
maybe expand to something like "As an eZ Engineer, I want to explore use cases identified to ... "
to refer to the actual developer use cases.